Linux 下的路径拼接不对

```
 dotnetCampus.TagToVersion
Current directory /home/runner/work/ant-design-blazor/ant-design-blazor
Args /home/runner/.dotnet/tools/.store/dotnetcampus.tagtoversion/1.0.6/dotnetcampus.tagtoversion/1.0.6/tools/netcoreapp3.1/any/dotnetCampus.TagToVersion.dll -t refs/tags/0.1.0-alpha01
Read tag version. Tag name is refs/tags/0.1.0-alpha01
Tag version is 0.1.0-alpha01
Find the version.props file. 
Auto finding version file
System.ArgumentException: Can not find /home/runner/work/ant-design-blazor/ant-design-blazor/build\Version.props
   at dotnetCampus.TagToVersion.Program.FindVersionFile(String versionFile) in D:\a\dotnetCampus.TagToVersion\dotnetCampus.TagToVersion\Program.cs:line 71
   at dotnetCampus.TagToVersion.Program.<>c.<Main>b__0_0(Program option) in D:\a\dotnetCampus.TagToVersion\dotnetCampus.TagToVersion\Program.cs:line 26
   at CommandLine.ParserResultExtensions.WithParsed[T](ParserResult`1 result, Action`1 action)
   at dotnetCampus.TagToVersion.Program.Main(String[] args) in D:\a\dotnetCampus.TagToVersion\dotnetCampus.TagToVersion\Program.cs:line 18
##[error]Process completed with exit code 255.
```

--- a/Program.cs
+++ b/Program.cs
@@ -59,7 +59,17 @@ namespace dotnetCampus.TagToVersion
             if (string.IsNullOrEmpty(versionFile))
             {
                 Console.WriteLine($"Auto finding version file");
-                versionFile = Path.GetFullPath(@"build\Version.props");
+                var buildFolder = Path.GetFullPath("build");
+                if (!Directory.Exists(buildFolder))
+                {
+                    buildFolder = Path.GetFullPath("Build");
+                    if (!Directory.Exists(buildFolder))
+                    {
+                        throw new ArgumentException($"Can not find build folder {buildFolder}");
+                    }
+                }
+
+                versionFile = Path.Combine(buildFolder, "Version.props");
 
                 if (!File.Exists(versionFile))
                 {

